tailwind.config.js

```diff
@@ -11,14 +20,20 @@ module.exports = {
                 blue: {
                     lighter: '#ECF5FF',
                     light: '#086BCF',
+                    lightTransparent: '#338AFF33',
+                    light300: '#338AFF',
                     DEFAULT: '#1165BA',
                     dark: '#014C97',
+                    dark200: '#003986',
```

Comment:
esse não é o formato ideal, depois podemos discutir melhor como vai funcionar a hierarquia de tokens com o time de design.
mas vamos deixar a estrutura de todos semelhante a do tailwind, que seria assim:

```suggestion
                    dark-200: '#003986',
```